src/stripe.common.ts

```diff
@@ -53,11 +53,10 @@ Stripe.HttpClient = HttpClient.HttpClient;
 Stripe.HttpClientResponse = HttpClient.HttpClientResponse;
 
 import CryptoProvider = require('./crypto/CryptoProvider');
-import EventEmitter = require('events');
 Stripe.CryptoProvider = CryptoProvider;
 
-import DefaultPlatformFunctions = require('./platform/DefaultPlatformFunctions');
-Stripe._platformFunctions = new DefaultPlatformFunctions();
+// @ts-ignore
+Stripe._platformFunctions = null;
```

Comment:
nit: do we need this setter at all?

Reply:
It's just to appease TypeScript 😅 I think functionally it makes more sense to remove this and add a few more `@ts-ignore`'s

Reply:
Do you have a preference?

Reply:
Let's keep it and try to clean up the entire pattern in a follow up